packages/grafana-ui/src/components/Tooltip/Popper.tsx

```diff
@@ -73,7 +73,12 @@ class Popper extends PureComponent<Props> {
                         className={`${wrapperClassName}`}
                       >
                         <div className={className}>
-                          {typeof content === 'string' ? content : React.cloneElement(content)}
+                          {typeof content === 'string' && content}
+                          {React.isValidElement(content) && React.cloneElement(content)}
+                          {typeof content === 'function' &&
```

Comment:
@hugohaggmark This change is sth that we discussed last week. Now, if the content passed to Popper is a function, it will be called with `PopperContentProps` interface arguments. Right now it enables only popper repositioning, but we can extend the API to enable showing/hiding popper as a result of an action inside popover.

Reply:
Ok, that would be awesome.